README.md

```diff
@@ -1,52 +1,39 @@
-
-[![Dependency Status](https://david-dm.org/actum/devstack.svg)](https://david-dm.org/actum/devstack)
-
 > Starter kit for universal full–fledged React apps. One stack for browser, mobile, server.
 
 > Based on [Este.js](https://github.com/este/este)
 
 ## Prerequisites
 
-- [node.js](http://nodejs.org) Node 6+
+- [node.js](http://nodejs.org) Node 6+, install via [nvm](https://github.com/creationix/nvm) highly recommended
 - [gulp](http://gulpjs.com/) `npm install -g gulp`
-- [git](https://git-scm.com/downloads) git cmd tool is required
-
-#### Optional
-
-- [Facebook SDK for iOS](https://developers.facebook.com/docs/ios/) In order to make Facebook login work on iOS
-- [Facebook SDK for Android](https://developers.facebook.com/docs/android/) In order to make Facebook login work on Android
-- [firebase-cli](https://firebase.google.com/docs/cli/) `npm install -g firebase-tools`
-- [firebase-bolt](https://github.com/firebase/bolt) `npm install -g firebase-bolt`
+- [yarn](https://yarnpkg.com/en/docs/install)
 - [react-native-cli](http://facebook.github.io/react-native/docs/getting-started.html) `npm install -g react-native-cli`
 
-If you are using different node versions on your machine, use [nvm](https://github.com/creationix/nvm) to manage them.
-
 ## Start Development
 
+- install dependencies `yarn`
+
+### Web app
 - run `gulp`
 - point your browser to [localhost:3000](http://localhost:3000)
-- build something beautiful
 
-React Native: [Getting Started](https://facebook.github.io/react-native/docs/getting-started.html)
+### iOS app
+- run `gulp ios`
+
+### Android app
+- run `gulp android`
 
 ## Dev Tasks
 
-- `gulp` run web app in development mode
-- `gulp ios` run iOS app in development mode
-- `gulp ios -p` run iOS app in production mode
-- `gulp android` run Android app in development mode
-- `gulp -p` run web app in production mode
-- `gulp -f` run web app in development mode, but only browser source rebuilds on file changes
-- `gulp jest` run jest tests
-- `gulp jest-watch` continuous test running for TDD
 - `gulp eslint` eslint
 - `gulp eslint --fix` fix fixable eslint issues
 - `gulp messages-extract` extract messages for translation
 - `gulp messages-check` check missing and unused translations
 - `gulp messages-clear` remove unused translations
-- `gulp favicon` create universal favicon
 - `npm run build-size-check` display info about latest build
 
+See [complete list of dev tasks](https://github.com/este/este#dev-tasks) on Este.
```

Comment:
please reference to fix revision
https://github.com/este/este/tree/af909fce150201a7355a52321f555df8af7994f5#dev-tasks